### PR TITLE
feat(base): add network support

### PR DESCRIPTION
**Pull request by JFrankfurt**

### CODE REVIEW

File: src/components/Menu/NetworkDropdown.tsx [hunk 1]
```diff
@@ -138,7 +138,6 @@ export default function NetworkDropdown() {
                         {n.name}
                       </TYPE.main>
                     </RowFixed>
-                    {n.blurb && <Badge>{n.blurb}</Badge>}
```
Comment (just-toby) [inline src/components/Menu/NetworkDropdown.tsx:141]: is this change intended?

**Review by just-toby — APPROVED** [review]

File: src/components/Menu/NetworkDropdown.tsx (hunk 1 above)
Comment (JFrankfurt) [reply src/components/Menu/NetworkDropdown.tsx:141]: yeah it'd dead code
